core/src/rust/filodb_core/src/index.rs

```diff
@@ -56,9 +57,22 @@ pub extern "system" fn Java_filodb_core_memstore_TantivyNativeMethods_00024_newI
             .open_or_create(directory.clone())?;
 
         let writer = index.writer::<TantivyDocument>(WRITER_MEM_BUDGET)?;
+
+        let mut merge_policy = LogMergePolicy::default();
+        merge_policy.set_del_docs_ratio_before_merge(0.1);
```

Comment:
I don't want to cause a config overload but is .1 a good number across to be hardcoded even in cases where there is high churn? Would frequent segment merges cause cpu spikes (Generally lesser segments will help with query performance)?

Reply:
I'm not sure we ever need to change this per instance so it feels like it's not something I'd put in a config file, though it can be converted to a config in the future.

This number only indicates the lower bound on when it is considered for merges.  It does not guarantee a merge at 10%, just that it will be compared to other files for merge candidates.  In general it only merges when there are many files that benefit from it.

Their logic is logarithmic for merging, so it tends to avoid constant merging.  In general I found the defaults besides this value were very sane and they recommend changing the delete ratio from 0 but don't provide a sensible default due to back compat.

My vote is to keep this at this point, monitor perf, and adjust later if needed.

Reply:
Thanks @rfairfax for the quick changes